src/Uno.Playground.Shared/App.xaml.cs

```diff
@@ -72,7 +72,11 @@ static void ConfigureFilters(ILoggerFactory factory)
 						// { "ReferenceHolder", LogLevel.Debug },
 					}
 				)
-				.AddConsole(LogLevel.Debug);
+#if DEBUG
```

Comment:
This line disables logging completely: https://github.com/nventive/Uno.Playground/blob/fae9785732cabb894e26596e3cfe007c927e3af5/src/Uno.Playground.Shared/App.xaml.cs#L38

Reply:
Reverted